### ISSUE TITLE
Show celebratory image upon reaching 100 days

### ISSUE THREAD

**Issue by lineaba** [lineaba/prog-o-meter#49]
__First timers only__
_This means that this issue have been determined a good place to start for someone who have not been contributing to open source before. Let us know how we can help you get started or if you have any questions. We would really love for you to join us_

This is a sub-part of issue #30, but I think it would be a very nice start. 
Wnen the user clicks the "1 more day" button upon completing their last day, let's display a nice celebratory image on the canvas. 

Here is a suggestion for how to solve it:
- Find a nice celebratory image (remember to find something with a creative common license or similar, so we are allowed to use it - or if you are creative, you could draw/create one for the project!). Store it in the same folder, as where you store the prog-o-meter.py on your computer
- Every time the user click "1 more day", check if this was there day 100
    - If this was the day 100, display image on canvas (you will have to decide on what the best location is, depending on the size of the image) [Here](http://effbot.org/tkinterbook/photoimage.htm) is the Tkinter guide on images, which might be helpful

As always, the above is only a suggestion, and there a probably many other ways to solve the same problem, so feel very free to take your own approach
Let me know if you have any questions, or would like any help getting started :)

**Comment by chiranjev** [comment]
I would like to take this issue.

**Comment by lineaba** [comment]
Hey, awesome, welcome to the project! I have been looking at your github repo, it looks like you have some experience coding. If you are interested, we have a little more challenging first-timer issue #24 which contains a few more steps to implement the code. You are very welcome to do either of them, I just wanted to point you in that direction! I am excited to have you contribute

**Comment by chiranjev** [comment]
Yes, I would like to take the issue #24 . Thanks for informing me about that.

**Comment by lineaba** [comment]
Nice! Welcome to the team, I'll add you as a contributor, and you can assign yourself to issue #24. Let me know if you have any questions :)

**Comment by lineaba** [comment]
@B-Souty The reason why I encouragesd chiranjev to take issue #24 was because this is a first-timers-only issue. I would like to keep it available for someone making their first PR. So would you mind choosing another issue? I was thinking about #44 Did we ever find a good cross-platform-solution to this?

**Comment by coa-coa** [comment]
May i take this one?

**Comment by lineaba** [comment]
@coa-coa I am sorry it took so long to get back to you, I have been out of time and studying for finals. But yes, if you are still interested, you can definitely take this! Let me know if you have any questions :)

**Comment by psuhas77** [comment]
If the issue is still open, can I take this one up?

**Comment by Komorebi-Girl** [comment]
Hi! If @coa-coa  isn't available, I am still pretty new to contributing to open-source projects and would be happy to take on this or any other task you recommend. Thank you!

**Comment by lineaba** [comment]
@Komorebi-Girl feel free to start working on this issue. We have not heard anything back from @coa-coa. 
One note that I am not sure whether was clear in the original description. Our project already have lots of different pop-up windows, and I think it is nice to not add others when we can avoid it. You can consider if adding the image fits best directly on the canvas (perhaps instead of the encouraging quote which is showed each time the user adds another day). Alternatively, the image could be added to the window that pops up and lets the user share their celebration on social media. Any of those options are good by me, so let me know what you think, or if you have other suggestions. 
And let me know if you have any questions.

**Comment by sourabh8244** [comment]
I would like to take this issue for my first PR. I hope to fit in the image in the canvas itself with an image that's open to use. May I start along side @Komorebi-Girl and this might just give me some experience in working on this even if it isn't as good as @Komorebi-Girl .

**Comment by Komorebi-Girl** [comment]
Thanks @lineaba! I'll come up with an idea of how I want to implement this feature and run it by you later today or tomorrow.

**Comment by lineaba** [comment]
@sourabh8244 You are welcome to work on this if you wanna get familiar with the project and tkinter. However, I plan on merging @Komorebi-Girl's contribution, since they "claimed" the issue first.
But I would highly encourage you play around with the code. Maybe you get ideas for something you think we should add, then just open a new issue and tell us about your idea, and you can start working on it. If you don't have any ideas yourself for things to do, take a look an see if there are any issues you would like to work on, which no one have claimed yet (there might not be any right now, since I haven't been updating recently, but keeping coming back and looking. Let me know if you have any questions, I'll be happy to help :)

**Comment by Komorebi-Girl** [comment]
Hi @lineaba, I'm sorry for this but can I surrender this issue to @sourabh8244? I was too excited to contribute to an open source project that I didn't realize that the project was written entirely in Python which I admittedly don't know well at all. I don't want to delay the project when there's someone ready and available to take this issue on. Again, sorry for the inconvenience.

**Comment by sourabh8244** [comment]
I would surely want to contribute if at all I can. I have already forked
your repository and going through your codes. I got to learn a lot. Should
I find an issue I feel like I can work upon I will certainly put forth my
name😊

**Comment by lineaba** [comment]
@Komorebi-Girl we are not in a hurry, so don't feel like you can't give this a try, we want this project to be a place for learning. Let me know if you would like some help finde a simpler issue to start out with. @sourabh8244 if you want to work on this issue go ahead and when you are ready, send a pull-request.

**Comment by RavinSG** [comment]
Hello, I'm looking for a OS project that I can contribute to as my initial step entering to the community. It would be great if I can take part in this! :)

**Comment by Vidy-a** [comment]
I can see this isn't assigned to anyone yet. Can I work on this? 
I have experience with python, but not with open source, and I'd love to start!

**Comment by import-keshav** [comment]
getting error while doing push request.
can anyone help me.
error is :: " Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists. "

**Comment by reedwilemon** [comment]
@lineaba Hi! I'm interested in contributing to this or other (simple) issues. Let me know! :)

**Comment by nisnt2411** [comment]
Hi  @lineaba 
if this issue is still open i would like to contribute it as my first PR , i'm good with python but a novice to open source. If in case it's not open still i would like you guys  to check my work, also it would be great if you guide me to some other beginner issues. let me know whatever case it is.

**Comment by sumit9696** [comment]
Can i take this issue as I am new to open source and would like to get my first PR merged so that I can work further with some confidence.?

**Comment by harsh7g** [comment]
If this issue still open I would like to start working on it

**Comment by pratheekcr** [comment]
Hi, I am new to open source and this issue would be a great way to start. Is this still open? Let me know, please.
